src/components/AppMapDetailsObj.ts

```diff
@@ -10,6 +10,7 @@ import ObjectInfo from '@/components/ObjectInfo';
 import { MapMgr, ObjectData, ObjectMinData, PlacementLink } from '@/services/MapMgr';
 import { MsgMgr } from '@/services/MsgMgr';
 import * as ui from '@/util/ui';
+const KUH_TAKKAR_ELEVATOR = 0x96d181a0;
```

Comment:
KUH_TAKKAR_ELEVATOR_HASH_ID, perhaps? Just to make it even more obvious what this is.